.github/workflows/release.yml

```diff
@@ -64,12 +64,12 @@ jobs:
           # linker = "arm-linux-gnueabihf-gcc"
           # EOF
       - name: Install rust target
-        run: rustup target add $TARGET
+        run: rustup target add "$TARGET"
       - name: Run build
         run: |
-          cargo build --release --verbose --target $TARGET
-          if [[ $RUNNER_OS == "Linux" ]]; then 
-            cargo appimage 
+          cargo build --release --verbose --target "$TARGET"
```

Comment:
Mostly, the linter complained about missing double quotes around expressions containing calls to variables.  These changes are obvious and do not break anything.